Documentation/ApiOverview/UpdateWizards/Concept/Index.rst

```diff
@@ -0,0 +1,33 @@
+.. include:: ../../../Includes.txt
+
+.. _update-wizards-concept:
+
+=============================
+The concept of update wizards
+=============================
+
+Update wizards are single PHP Classes which provide an automated way to update certain
+parts of a TYPO3 installation. There parts might be database or file system related.
+
+Wizards should be provided to ease updates for integrators and administrators. They
+are an addition to the database migration which is handled by the Core based on
+:file:`ext_tables.sql`.
+
+The execution order is not defined. Each administrator is able to execute wizards and
+migrations in any order. They are also completely skippable.
+
+Each wizard is able to check pre-conditions to prevent execution, if nothing has to
+be updated. The wizard can log information and executed SQL statements, which can be
+displayed after execution.
+
+Best practice
+=============
+
+Each extension can provide as many update wizards as necessary. Each one should do
```

Comment:
"[...] necessary. Each **wizard** should [...]"

* specify "wizard" to prevent confusion with "extension"

Reply:
wizard in lowercase? As above Wizards has to be uppercase.

Reply:
Yes, in lowercase. The uppercase was a mistake on my part. Sorry for that!